cypress/page/shopping-cart.page.ts

```diff
@@ -4,11 +4,11 @@ class ShoppingCartPage {
   private proceedToCheckOutLogInButton: string;
 
   constructor() {
-    this.proceedToCheckOutLogInButton = ".cart_navigation span";
+    this.proceedToCheckOutLogInButton = "[title='Proceed to checkout']";
   }
 
   public proceedToCkeckoutLogin(): void {
-    cy.get(this.proceedToCheckOutLogInButton).click();
+    cy.get(this.proceedToCheckOutLogInButton).eq(1).click();
```

Comment:
you should not need the call to the `eq` command, if there are more than one HTML tag with the same title attribute, try to add something else to the selector. Example:
`.cart_navigation [title='Proceed to checkout']`

Reply:
Thanks holmes